runner/src/server/plugins/logging.ts

```diff
@@ -1,4 +1,70 @@
-import pluginLogging from "../../../../digital-form-builder/runner/src/server/plugins/logging";
+import pino from "hapi-pino";
+import {config} from "./utils/AdapterConfigurationSchema";
+
+export default {
+    plugin: pino,
+    options: {
+        prettyPrint: false,
+        level: config.logLevel,
+        base: null,
+        formatters: {
+            level: (label) => {
+                return {severity: label.toUpperCase()}; // Align severity levels for CloudWatch
+            },
+        },
+        messageKey: "message",
+        customProps: (request) => ({
+            requestId: request.info.id, // Include request ID
+            userAgent: request.headers["user-agent"], // Add user agent for context
+            route: request.route.path, // Include route information
+        }),
+        serializers: {
```

Comment:
If serializers run after hooks, then this makes the inclusion of serializers here a... fallback? Given that req and res are deleted in the hook?